server/views/pages/edit/heightMetric.njk

```diff
@@ -4,17 +4,12 @@
 {% from "govuk/components/input/macro.njk" import govukInput %}
 {% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
 {% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from '../../components/miniBanner/miniBanner.njk' import miniBanner %}
 {% set hideBackLink = true %}
 {%
   set breadCrumbs = [
-  {
-  text: 'Digital Prison Services',
-  href:  "/" | prependBaseUrl
-  },
-  {
-  text: breadcrumbPrisonerName,
-  href: "/prisoner/" + prisonerNumber
-  }
+    { text: 'Digital Prison Services', href:  "/" | prependBaseUrl },
+    { text: breadcrumbPrisonerName, href: "/prisoner/" + prisonerNumber }
   ]
```

Comment:
"Personal" breadcrumb is missing and there is no Cancel option

Same for the Imperial version of the page